src/pages/TopUp/index.tsx

```diff
@@ -114,11 +116,6 @@ const _TopUpPage: React.FC<Props> = () => {
               setValidators([]);
               setLoading(false);
               return;
-            }
-
-            if (response.length === 0) {
-              setValidators([]);
-              setLoading(false);
             } else {
               // query by public keys
               const pubKeysCommaDelineated = `${response
```

Comment:
Side note, is there a reason we're chopping this list at 50 items? Is this due to limitations on the beaconcha.in api call below?